kolibri/plugins/facility/assets/src/views/users/sidePanels/EnrollLearnersSidePanel.vue

```diff
@@ -186,29 +180,13 @@
 
       // Methods
       async function setClassUsers() {
-        classLearners.value = [];
-        classCoaches.value = [];
         loading.value = true;
-        const membershipsPromise = MembershipResource.fetchCollection({
-          getParams: { user_ids: Array.from(props.selectedUsers).join(',') },
-        });
-        const userModelsPromise = FacilityUserResource.fetchCollection({
-          getParams: {
-            by_ids: Array.from(props.selectedUsers).join(','),
-          },
-        });
         try {
-          const [membershipsData, userModels] = await Promise.all([
-            membershipsPromise,
-            userModelsPromise,
-          ]);
-          membershipsByUser.value = groupBy(membershipsData, 'user');
-          classLearners.value = Object.keys(membershipsByUser.value);
-          classCoaches.value = Array.from(
-            userModels
-              .filter(user => user.roles.some(role => role.kind.includes(UserKinds.COACH)))
-              .reduce((set, user) => set.add(user.id), new Set()),
-          );
+          const classMemberships = await MembershipResource.fetchCollection({
+            getParams: { user_ids: Array.from(props.selectedUsers).join(',') },
```

Comment:
Seems like we will still need the `force` prop set to true here? If I remove a user from a class, and then I want to enroll them again, it wont because in the previous cached response the user was enrolled:

https://github.com/user-attachments/assets/f79a6302-e848-442c-a645-20cbe17f536b